### ISSUE TITLE
Audio Features render same results

### ISSUE THREAD

**Issue by karass2** [Marekkon5/onetagger#163]
Hi there!

I've tried to process 1300 songs, 800+ of which have been succesful, but I look at the results, it seemed that one tagger has labeled all of the songs with the same tags. See the image bellow: Comment 2 column shows real Energy, Danceability, Valency and Popularity values 
 for those tracks. 

![image](https://user-images.githubusercontent.com/46132879/205747245-6b8f4f81-c9e5-45d1-820f-d163efe043ca.png)

This second image shows my Audio Features settings. Am I doing something wrong?

![image](https://user-images.githubusercontent.com/46132879/205747388-f83ca143-501a-422f-9508-5202b21a8591.png)

**Comment by bascurtiz** [comment]
Hi!
Your Threshold sliders reveal why...
![image](https://user-images.githubusercontent.com/34075088/205749895-299e69bf-3f2d-4e22-bc3c-e5521facb7ca.png)
This way all is set to write #-med(ium) when it's between 0 and 100 value.

The default values are shown in Help of Audio Features, page 2/2 (which I recommend to set them back to).

Hope the picture below makes more sense to you!
![image](https://user-images.githubusercontent.com/34075088/205751981-caf28ab7-354f-4abb-b767-7af171770101.png)

Lemme know if you got any more questions.

**Comment by karass2** [comment]
Yes, now that makes a lot of sense. Thanks a lot for clearing that up.

One suggestion there. It would be nice if users were able to define how they want to label the results:
Just to make the column a bit smaller and easily readable, I would love to be able to mark it as L,M,H for high, low, medium and E,D,V,P, for Energy, Danceability, Valency and Popularity. As a result, one track could have a record of E-L, D-M, V-N/P, P-P/U.

I hope this makes sense. 

Thanks again for the amazing work you guys are doing. This app is already better then most of the stuff that's out there!
I do have some other comments about other modules, but that's another story for another post (coming soon).

**Comment by bascurtiz** [comment]
Great.
A total user-defined approach would be overkill and complex GUI-wise, however, I do like your abbreviations!
I can imagine we could make a switch between default i.e. _(#dance-high)_ and short notation _(#D-H)_.
I will discuss this with @Marekkon5.
Thanks for the heads up!
Feel free to make another post for suggestions on other functions.
